index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/navbar.css">
```

Comment:
Will you be later using your component library to build the e-commerce site? (as mentioned in lms)

Reply:
Yes I am using it

Reply:
Okay, You can reference it either in your main stylesheet or in index.html and reuse the components :)

Reply:
I forgot completely about it🤦‍♂️, Thanks!

Reply:
no problem!

Reply:
I have not created nav bar in component library, I'll add and later use it